src/utils/parseSelector.js

```diff
@@ -2,7 +2,7 @@ import postCssSelectorParser from "postcss-selector-parser";
 
 export default function(selector, result, node, cb) {
   try {
-    postCssSelectorParser(cb).process(selector);
+    postCssSelectorParser(cb).processSync(selector);
```

Comment:
We could make this fully match the code from stylelint and make add a return: `return postCssSelectorParser(cb).processSync(selector)`

Reply:
ping @xboy2012

Reply:
Sorry for late response.

I was working on an old version of stylelint, in which I see no `return` in the source code.

I read the latest code of stylelint again. I see the `return` there. My PR is Updated.

Reply:
For convenience, I post the source code url here

https://github.com/stylelint/stylelint/blob/060310cfd6383175ee06fe4d15aee27a4e202671/lib/utils/parseSelector.js